### ISSUE TITLE
Integrate  human annotations with AI generated extractions

### ISSUE THREAD

**Issue by palanik1** [eBPFDevSecTools/ebpf-projects-annotations#16]

**Comment by sdsen** [comment]
@palanik1 will this get addressed with you query based script? If so can you write the process in the right readme and close this issue?

**Comment by palanik1** [comment]
Depends on #47

**Comment by palanik1** [comment]
Duplicate of #47
